documentation/design-docs/dotnet-tools.md

```diff
@@ -663,6 +664,53 @@ Examples:
     Wrote 12576 bytes to file
     Complete
 
+REPORT
+
+    dotnet-gcdump report -f|--file <input_dump_path>
+    
+    Generate report into stdout from a previously generated gcdump or from a running process.
+    
+    Usage:
+      dotnet-gcdump report [options]
+    
+    Options:
+      -f, --file        The file to read gcdump from.
```

Comment:
Earlier I had mentioned in a comment:

> dotnet-dump analyze treats the input file as a positional parameter rather than an option and I think it would be worthwhile to be consistent on that part of the behavior. It also makes the command line a little more compact.

Not sure if it got lost in the noise or there was a specific objection to it? That would change the syntax to:

dotnet-gcdump report [-p <pid>] [-t type] [<gcdump_filename>]

Reply:
Indeed it got lost 😃 I'll update the PR with this. Consistency is important.